haskell/tools/ghc_wrapper.py

```diff
@@ -39,7 +37,9 @@ def main():
 
     cmd = [args.ghc] + ghc_args
 
-    subprocess.check_call(cmd)
+    returncode = subprocess.call(cmd)
+    if returncode != 0:
+        return returncode
```

Comment:
Doesn't this require something like `sys.exit(main())` in the `if __name__ == "__main__"` part?